app/code/AKisilenko/ModuleLesson11/Model/FileList.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AKisilenko\ModuleLesson11\Model;
+
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+
+/**
+ * Class MyFiles
+ * @package file_list
+ */
+class FileList
+{
+    public function makeFileList()
+    {
+        $path = realpath('/misc/apps/kalexey/app/code/');
```

Comment:
This directory could be different in other users. 
Please inject \Magento\Framework\Filesystem\DirectoryList $directoryList
to this class and use $path = realpath($this->directoryList->getRoot() . '/app/code/');
to get the proper path